data/undergrad.yaml

```diff
@@ -327,22 +327,22 @@ Single Variable Real Analysis:
     Taylor's theorem with integral form for remainder:
     Taylor's theorem with Lagrange form for remainder:
     Taylor series expansions:
-  Usual functions (trigonometric, rational, $\exp$, $\log$, etc):
+  Elementary functions (trigonometric, rational, $\exp$, $\log$, etc):
     polynomial functions: 'polynomial.eval'
     rational functions:
     logarithms: 'real.log'
     exponential: 'real.exp'
     power functions: 'monoid.pow'
-    circular trigonometric functions: 'real.sin'
+    (circular) trigonometric functions: 'real.sin'
     hyperbolic trigonometric functions: 'real.sinh'
-    reciprocal circular trigonometric functions: 'real.arcsin'
-    reciprocal hyperbolic trigonometric functions:
+    inverse (circular) trigonometric functions: 'real.arcsin'
+    inverse hyperbolic trigonometric functions:
   Integration:
     integral over a segment of piecewise continuous functions:
     antiderivatives:
     Riemann sums:
     antiderivative of a continuous function:
-    usual methods to compute integrals:
+    the usual methods to compute integrals:
```

Comment:
Actually I think this was meant as a subheading for the next two items so we should remove that line.